orchagent/portsorch.cpp

```diff
@@ -7607,7 +7620,7 @@ void PortsOrch::generateQueueMapPerPort(const Port& port, FlexCounterQueueStates
             {
                 continue;
             }
-            queueTypeVector.emplace_back(id, queueType);
+            queueTypeVector.emplace_back(id, sai_queue_type_string_map[queueType]);
```

Comment:
For the queue counters, could you check if this bulk counter poll includes all the types including WRED ECN that got added recently?

Reply:
yes. my last commit adjust WRED ECN accordingly. https://github.com/sonic-net/sonic-swss/pull/3504/commits/e8fc577cfa66ad8058ca154a3b81c0c020a63745

Reply:
Thanks @stephenxs